### PR TITLE
added seek command

### PR DESCRIPTION
**Pull request by benno1237**
Added a simple seek command.
Changes in this PR have been tested and are working.

### CODE REVIEW

File: audio/cog/commands/hybrid_commands.py [hunk 1]
```diff
@@ -529,6 +531,105 @@ async def command_volume(self, context: PyLavContext, volume: Range[int, 0, 1000
...
+                mn = int(match.group(2)) if match.group(2) else 0
+                sec = int(match.group(3)) if match.group(3) else 0
+                pos = sec + (mn * 60) + (hr * 3600)
+                seek_ms = pos * 1000
+            else:
+                seek_ms = 0
+
+            await context.send(
+                embed=await context.lavalink.construct_embed(
+                    description=_("Moved to {time}.").format(time=format_time(seek)),
```
Comment (Drapersniper) [inline audio/cog/commands/hybrid_commands.py]: please make use of the messageable kwarg of the messageable and pass the context to it (this ensures we are using the bot colour config, on all your construct_embed calls

File: audio/cog/commands/hybrid_commands.py [hunk 2]
```diff
@@ -529,6 +531,105 @@ async def command_volume(self, context: PyLavContext, volume: Range[int, 0, 1000
...
+                ),
+                ephemeral=True,
+            )
+
+        except Exception as e:
+            print(e)
```
Comment (Drapersniper) [inline audio/cog/commands/hybrid_commands.py]: use the Logger - in any case, if it raises an exception do we want to follow through with the seek op?

**Review by Drapersniper — CHANGES_REQUESTED** [review]

**Comment by benno1237** [comment]
I have read the CLA Document and I hereby sign the CLA

File: audio/cog/commands/hybrid_commands.py (hunk 2 above)
Comment (benno1237) [reply audio/cog/commands/hybrid_commands.py]: I didnt want this to be in there. Sorry

File: audio/cog/commands/hybrid_commands.py [hunk 3]
```diff
@@ -529,6 +531,112 @@ async def command_volume(self, context: PyLavContext, volume: Range[int, 0, 1000
             ephemeral=True,
         )
 
+    @commands.hybrid_command(name="seek", description="Seek the current track.")
+    @commands.guild_only()
+    @requires_player()
+    async def command_seek(self, context: PyLavContext, seek: str):
+        """Seek the current track.
+
+        seek can either be a number of seconds or a timestamp
```
Comment (Drapersniper) [inline audio/cog/commands/hybrid_commands.py]:
```suggestion
        Seek can either be a number of seconds or a timestamp.
```

**Pull request merged**